packages/cdktf-cli/bin/cmds/helper/terraform-cloud-client.ts

```diff
@@ -54,22 +54,30 @@ async function get(url: string, token: string) {
 async function post(url: string, token: string, data: string) {
     return new Promise<any>((ok, ko) => {
         const req = https.request(format(url), { 
-            headers: { 'Authorization': `Bearer ${token}`,
+            headers: {
+                'Authorization': `Bearer ${token}`,
                 'Content-Type': 'application/vnd.api+json',
-                'Content-Length': data.length }, 
-            method: 'POST' }, res => {
-                if (res.statusCode) {
-                    const statusCode = res.statusCode;
-                    if (!VALID_ERROR_CODES.includes(statusCode)) {
-                        return ko(new Error(res.statusMessage));
-                    }
-                }
+                'Content-Length': data.length
+            },
+            method: 'POST'
+        }, res => {
             const data = new Array<Buffer>();
             res.on('data', chunk => data.push(chunk));
-
             res.once('error', err => ko(err));
             res.once('end', () => {
                 const response = JSON.parse(Buffer.concat(data).toString('utf-8'));
+
+                if (res.statusCode) {
+                    const statusCode = res.statusCode;
+                    if (!VALID_ERROR_CODES.includes(statusCode)) {
```

Comment:
I know `VALID_ERROR_CODES` was there before, but the naming confuses me a lot. Since we're touching this now, could you change this to something more descriptive, please?

Reply:
done 👍